### ISSUE TITLE
Remove "fbjs" dependency

### ISSUE THREAD

**Issue by the-spyke** [grommet/grommet#4126]
### Expected Behavior

No excess or outdated dependencies. No warnings on install.

### Actual Behavior

```
warning grommet > recompose > fbjs > core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

Depending on `fbjs` is too much, not mentioning it pulls some outdated deps like `core-js@1`. I thinks `recompose` is not that important in `the age of hooks` and can be refactored out together with `fbjs`.

### Steps to Reproduce

1. Install grommet

### Your Environment

- Grommet version: 2.13.0
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by the-spyke** [comment]
There're also a bunch of warnings with `css` dependency:

```
warning grommet > css > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning grommet > css > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning grommet > css > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
```

**Comment by ShimiSun** [comment]
@the-spyke these warnings were fixed as part of #4083, it wasn't officially released yet, the fix is currently on the [stable](https://github.com/grommet/grommet/wiki/What-is-grommet-stable-and-how-to-use-it%3F) branch until next official release.

**Comment by the-spyke** [comment]
Fixed in `grommet@2.14.0`

**Comment by ShimiSun** [comment]
Thanks for the follow-up 💟
